app/controllers/course/user_invitations_controller.rb

```diff
@@ -164,7 +164,7 @@ def aggregate_errors
   # @return [Array<String>]
   def invalid_course_user_errors
     invalid_course_users.map do |course_user|
-      user = self.class.helpers.display_user(course_user.user || course_user)
+      user = self.class.helpers.display_course_user(course_user.user || course_user)
```

Comment:
This was to [fix a failing spec here](https://github.com/Coursemology/coursemology2/blob/9f4d661480af367056a165f68b3ff982274657d5/spec/controllers/course/user_invitations_controller_spec.rb#L83-L100). 

- Will look into seeing whether I can just modify the spec to fix this. 
- Should we just use `course_user` as the param? I think it would be easier.